src/platform/forms-system/src/js/review/submit-states/GenericError.jsx

```diff
@@ -9,11 +9,11 @@ import FormSaveErrorMessage from 'platform/forms/components/review/FormSaveError
 import { Column, Row } from 'platform/forms/components/common/grid';
 // const FormSaveErrorMessage = props => props?.children;
 export default function GenericError(props) {
-  const { appType, formConfig, renderErrorMessage, onSubmit, testId } = props;
+  const { appType, formConfig, onSubmit, testId } = props;
   let submitButton;
   let submitMessage;
 
-  if (renderErrorMessage) {
+  if (formConfig.submissionError) {
```

Comment:
The prop `renderErrorMessage` was coming in undefined so I went ahead and checked for  `submissionError` in formConfig since that is the hook that is supposed to overwrite the generic error message. 

Not sure what `renderErrorMessage` is supposed to do but it is being passed as a bool up top and is undefined in this file for whatever reason.

Origin of `renderErrorMessage` = https://github.com/department-of-veterans-affairs/vets-website/blob/LinnJS-remove-sign-in-1010cg/src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx#L100

Reply:
See the main comment for this review for a bit more context.